apps/happy-blocks/block-library/universal-header/header.scss

```diff
@@ -167,6 +167,37 @@ body.archive {
 			padding: 0 24px;
 		}
 	}
+
+	&.happy-blocks-header__learn {
+		margin-top: 120px;
+		margin-bottom: 70px;
+
+		.happy-blocks-global-header-site__title {
+			display: flex;
+			flex-direction: column;
+
+			h1 {
+				/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+				font-size: 70px;
```

Comment:
Could we use `rem` instead and get rid of the `stylelint-disable`? Or are there side-effects?

Also, we could already address mobile 💪

Reply:
Thank you for sharing that we can do `rem(unitPX);` that allowed me to have `rem(70px)`